portfolio/src/main/webapp/index.html

```diff
@@ -18,7 +19,7 @@
         <li><a href="#about" onclick="menuToggle()">About</a></li>
         <li><a href="#works" onclick="menuToggle()">Works</a></li>
         <li><a href="#contact" onclick="menuToggle()">Contact</a></li>
-        <li><a href="https://zety.com/mycv/NateyanaBlake" target="_blank">Resume</a></li>
+        <!-- <li><a href="https://zety.com/mycv/NateyanaBlake" target="_blank">Resume</a></li> -->
```

Comment:
Why comment this line?

Reply:
Changed my mind about including a resume link

Reply:
Ok. If not needed, can you delete this line?

Reply:
Sure